compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -13522,7 +13570,11 @@ OMR::Z::TreeEvaluator::primitiveArraycopyEvaluator(TR::Node* node, TR::CodeGener
          {
          bool evaluatedLengthMinusOne = false;
 
-         byteLenReg = cg->evaluateLengthMinusOneForMemoryOps(byteLenNode, true, evaluatedLengthMinusOne);
+         // Don't evaluate length minus one if byteLenNode was already evaluated
+         if (!alreadyEvaluatedByteLenNode)
```

Comment:
Lets stay consistent with `evaluatedLengthMinusOne` and name this `evaluatedByteLenNode`.